ablySpec/RealtimeClientConnection.swift

```diff
@@ -1302,6 +1302,54 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15d
+                pending("should recover from disconnection and messages delivered once the connection is resumed") {
```

Comment:
Actually, I may have misread RTN15d and this suggested change is wrong. However, is this test then not the same as RTN15a, I'm not sure why there are two different tests?